### PR TITLE
Feature/008 advanced testing mode

### PR DESCRIPTION
**Pull request by pkerntke**

### CODE REVIEW

File: app/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+# Default ignored files
+/workspace.xml
```
Comment (emreakca) [inline app/.idea/.gitignore:3]: why

File: app/app/.idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```
Comment (emreakca) [inline app/app/.idea/.gitignore]: do

File: app/app/.idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <codeStyleSettings language="XML">
+      <indentOptions>
```
Comment (emreakca) [inline app/app/.idea/codeStyles/Project.xml]: we

File: app/app/.idea/gradle.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (emreakca) [inline app/app/.idea/gradle.xml]: need

File: app/app/.idea/misc.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/build/classes" />
+  </component>
```
Comment (emreakca) [inline app/app/.idea/misc.xml]: these

File: app/app/.idea/modules.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
```
Comment (emreakca) [inline app/app/.idea/modules.xml]: .idea

File: app/app/.idea/vcs.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (emreakca) [inline app/app/.idea/vcs.xml]: files?

File: app/app/local.properties [hunk 8]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (emreakca) [inline app/app/local.properties]: `## This file must *NOT* be checked into Version Control Systems,`

File: app/app/local.properties [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Wed May 13 07:59:04 CEST 2020
+sdk.dir=/home/philip/Android/Sdk
```
Comment (emreakca) [inline app/app/local.properties]: ...

Comment (pkerntke) [reply app/app/local.properties]: sry no idea how they get into my commit

**Review by emreakca — CHANGES_REQUESTED** [review]
Remove .idea and local.properties
otherwise looks fine

File: app/app/local.properties (hunk 9 above)
Comment (emreakca) [reply app/app/local.properties]: Np, happens ^^

**Review by emreakca — APPROVED** [review]

**Pull request merged**